wiki/Updating.md

```diff
@@ -9,6 +9,6 @@ Run `docker compose pull screeps` to download any new version of the image.
 You can run `docker compose pull` without the `screeps` specifier to pull all new images for the compose file.
 
 ### Update mods and bots
-Those that have specific versions set in `config.yml` can be updated by changing that.
-To update mods/bots that was installed without a version requirement (defaults to @latest at time of first startup) will have to be updated from within the container.
-See [Issue #23](https://github.com/Jomik/screeps-server/issues/23)
\ No newline at end of file
+Mods and bots can be updated by running `npm run update` and restarting the server. This will go over the
```

Comment:
```suggestion
Mods and bots can be updated by running `docker compose exec screeps start --update` and restarting the server. This will go over the
```

The suggested installation for this server doesn't include you cloning the repo. If you see the way I use it [here](https://github.com/Jomik/screeps-ai/blob/main/docker-compose.yml), it is purely the config.yml and docker-compose.yml

Simply because it does not need npm and shouldn't interfere with your bot setup 😄